src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/HttpResourceManagerRpc.java

```diff
@@ -301,4 +311,66 @@ public Policy replacePolicy(String projectId, Policy newPolicy) throws ResourceM
       throw translate(ex);
     }
   }
+
+  @Override
+  public void clearOrgPolicy(String resource, ClearOrgPolicyRequest request) {
+    try {
+      resourceManager.folders().clearOrgPolicy(resource, request).execute();
+    } catch (IOException ex) {
```

Comment:
IOExceptions should not be converted into runtime exceptions

Reply:
ping

Reply:
done @elharo PTAL